spec/intergration/user_show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Users index', type: :system do
+  describe 'users#show page' do
+    before(:each) do
+      @user1 = User.create(name: 'Sharon',
```

Comment:
**In your pshow page add**
- [ ] Tests that redirects to the post's show page when you click on a user's post.
- [ ] Tests that redirects to the user's post's index page when you click on a **see all posts**.
- [ ]